scripts/session.js

```diff
@@ -134,13 +135,34 @@ ws.onmessage = function(evt){
       this.className = 'hidden-point';
     });
     Cookies.remove('point');
+    $('#statistics').hide();
+    $('.point_count').remove();
   }
 
   // show all votes
   if(my_received_message.type == "show_all_votes"){
+    var pointArray = [];
     $("td[id]").each(function(){
       this.className = '';
+      if(this.innerText != '') {
+        pointArray.push(this.innerText);
+      }
+    });
+
+    var pointHash = {};
+    pointArray.forEach(function(point_value){
+       if(!pointHash[point_value]) {
+        pointHash[point_value] = 1;
+       } else {
+        pointHash[point_value] += 1;
+       }
     });
+
+    for ( var key in pointHash) {
```

Comment:
`for...in` is OK here since there are no any prototypes on `pointHash`. But according to best practise, it's better to use [`Object.keys`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys).